resources/store/sessionStore.ts

```diff
@@ -0,0 +1,19 @@
+import create from 'zustand';
+
+export interface SessionInterface {
+    token: string | null,
```

Comment:
Codacy has a fix for the issue: [Replace `····token:·string·|·null,` with `··token:·string·|·null;`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
  token: string | null;
```